src/index.html

```diff
@@ -2,11 +2,391 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="
+      width=device-width,
+      initial-scale=1.0
+      "
+    >
```

Comment:
```suggestion
    <meta
      name="viewport"
      content="width=device-width, initial-scale=1.0"
    >
```
Мені здається краще зробити так